UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.API.FunctionalTests/Helper/TestConfiguration.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.Extensions.Configuration;
+
+namespace UKHO.ExternalNotificationService.API.FunctionalTests.Helper
+{
+    public class TestConfiguration
+    {
+        protected IConfigurationRoot ConfigurationRoot;
+        public string ensApiBaseUrl;
+        public string payloadFolder;
+        public string payloadFileName;
+
```

Comment:
Please make use of automatic properties: 
public string EnsApiBaseUrl { get; set; }
public string PayloadFolder  { get; set; }
public string PayloadFileName  { get; set; }

Reply:
Done required modification.